client/components/accordion/style.scss

```
@@ -182,7 +182,7 @@ $accordion-background-expanded: $white;
 		color: var( --color-warning );
 
 		&:hover {
-			color: lighten( $alert-yellow, 5% );
+			color: var( --color-warning-light );
```

```suggestion
			color: var( --color-warning );
```

Let's do this instead for better contrast

---

Actually, I see now it's a hover color. Instead, let's just remove the hover styles on all of the accordion status icons. I just tried it out and the slight color change on the icons are barely noticeable.

---

done in eeabaa4